src/main/archetype/ui.content/src/main/content/jcr_root/conf/__confFolderName__/settings/wcm/templates/experience-fragment-page/structure/.content.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:deviceGroups="[mobile/groups/responsive]"
+        cq:lastModified="{Date}2018-10-17T10:18:24.970+02:00"
+        cq:lastModifiedBy="admin"
+        cq:template="/conf/${confFolderName}/settings/wcm/templates/experience-fragment-venia-demo-store"
```

Comment:
there is still "venia" hardcoded in the demo-store name

Reply:
Should be `/conf/${confFolderName}/settings/wcm/templates/experience-fragment-page`?

Reply:
Arrrgh, yes it should, I missed this one...